qt_ui/windows/newgame/QNewGameWizard.py

```diff
@@ -457,6 +461,7 @@ def on_campaign_selected():
             else:
                 timePeriodPreset.setChecked(True)
             self.advanced_iads.setEnabled(campaign.advanced_iads)
+            self.iads_label.setEnabled(campaign.advanced_iads)
```

Comment:
Don't you need to fix the styling here?

Reply:
No, all I need to do is disable the label, look at line 398...